OCPBUGS-44063: Removing RHEL 9.2 update requirement 4.15+ (#86171)
openshift/openshift-docs: modules/installation-vsphere-infrastructure.adoc

|Optional: Networking (NSX-T)
|vSphere 7.0 Update 2 or later, or VMware Cloud Foundation 4.3 or later; vSphere 8.0 Update 1 or later, or VMware Cloud Foundation 5.0 or later
|For more information about the compatibility of NSX and {product-title}, see the Release Notes section of VMware's link:https://docs.vmware.com/en/VMware-NSX-Container-Plugin/index.html[NSX container plugin documentation].

> |CPU micro-architecture

Reviewer: Do we really need to remove this part? Because this doc is for fresh installation, I think this section can guide customers to choose a proper CPU architecture. WDYT?

Reviewer 2: hi @JianLi-RH , according to @wking in https://issues.redhat.com/browse/OCPBUGS-44063 this information is only relevant for the 4.12-4.13 versions and no longer applies as of 4.15. WDYT?

Reviewer: Maybe we can remove the version numbers from this section, and only list the expected ISA? @wking WDYT?

Reviewer 3: I'm not installer/CPU enough to be strongly opinionated here.  But I'm naively in favor of pointing out that there are constraints, and maybe there are RHEL-side docs we can link about what the minimum requirements are for the current RHEL version?  The installer/RHCOS docs folks would have to have some mechanism to bump those links when the base RHEL version changed though, instead of linking to an old KCS that is opening with discussion of etcd operator conditions.

Reviewer: @wking Agree that. As a cluster administrator, I think I'd like to know the minimum requirements but a old KCS. So link a RHCOS would be better.

Reviewer 4: FYI, I left a [comment](https://github.com/openshift/openshift-docs/pull/86171/files#r1883377806) in `modules/installation-minimum-resource-requirements.adoc`

Author: Thanks, all :) applied feedback (@bscott-rh and I discussed offline). Feel free to re-verify!

|CPU micro-architecture
|x86-64-v2 or higher

> |OpenShift 4.13 and later are based on RHEL 9.2 host operating system which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Microarchitecture requirements documentation].

Reviewer 5: I know this is existing text, but is this change worth it?

```suggestion
|{product-title} version 4.13 and later are based on RHEL 9.2 host operating system which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Microarchitecture requirements documentation].
```

> |{product-title} version 4.13 and later are based on RHEL 9.2 host operating system which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Microarchitecture requirements documentation].

Reviewer 6: ```suggestion
|{product-title} version 4.13 and later are based on the {op-system-base} 9.2 host operating system, which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Microarchitecture requirements documentation].
```

Reviewer 6: For 4.16 and 4.17, RHEL 9.4 is the host operating system, I think: https://access.redhat.com/articles/6907891

Author: Ok! I'll reach out to the install writers and see if this is something they'd like to update :) Thanks @dfitzmau !

CC @bscott-rh @skopacz1 👀 WDYT?

Author: Got confirmation from Seb that this should be covered in a separate story, so I'll create the Jira ticket and assign it to Brendan Daly. We can move forward with this PR

> |{product-title} version 4.13 and later are based on the {op-system-base} 9.2 host operating system, which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Microarchitecture requirements documentation].

Reviewer 6: ```suggestion
|{product-title} version 4.13 and later are based on the {op-system-base} 9.2 host operating system, which raised the microarchitecture requirements to x86-64-v2. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[Architectures] 
 in the {op-system-base} documentation.
```
